src/EasyAbp.FileManagement.Web/Pages/FileManagement/Files/File/ReUploadModal.cshtml.cs

```diff
@@ -50,14 +51,15 @@ public virtual async Task<IActionResult> OnPostAsync()
                 throw new ReUploadWithDifferentExtensionException();
             }
 
-            var updateFileDto = new UpdateFileInput
+            var updateFileDto = new UpdateFileWithStreamInput
             {
-                FileName = dto.FileName,
-                MimeType = UploadedFile.ContentType,
-                Content = await UploadedFile.GetAllBytesAsync()
+                Content = new RemoteStreamContent(UploadedFile.OpenReadStream(), UploadedFile.FileName)
```

Comment:
I think `UploadedFile.FileName` should be changed back to `dto.FileName`.

Reply:
Yes, you are right.